### ISSUE TITLE
Proposal: (Rich)TextBlock should support triple-click to select complete text/paragraph

### ISSUE THREAD

**Issue by Felix-Dev** [microsoft/microsoft-ui-xaml#2128]
# Proposal: (Rich)TextBlock should support triple-click to select complete text/paragraph

## Summary
(Rich)TextBlocks with text selection enabled (via [IsTextSelectionEnabled](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textblock.istextselectionenabled#Windows_UI_Xaml_Controls_TextBlock_IsTextSelectionEnabled)) should have support for the user to triple-click to select the complete text/paragraph. Currently, only single words can be selected with clicks, no matter how many clicks are "chained" by the user.

Sample app demonstrating current behavior included below.

## Rationale
The (Rich)TextBlock control should provide a text selection experience matching that found on the web and in the UWP TextBox control (for example) and thus expected by users.

## Scope
| Capability | Priority |
| :---------- | :------- |
| (Rich)TextBlock has triple-click support to select the complete text/paragraph. | Must |

## Attached Files
[TextBlockSelection.zip](https://github.com/microsoft/microsoft-ui-xaml/files/4351383/TextBlockSelection.zip)

**Comment by MarcAnt01** [comment]
@chris-han can you modify the issue also to include RichTextBlock, please?

**Comment by Felix-Dev** [comment]
@MarcAnt01 Already in the process of editing it, will let you know once finished editing. (Also, you pinged the wrong person 😁)

**Comment by MarcAnt01** [comment]
Yeah, I meant to mention Chris (but looks like he isn't the only Chris here) since he was the one who closed my issue, anyway thank you for doing it

**Comment by MarcAnt01** [comment]
Can this be reopened please?

**Comment by MartinZikmund** [comment]
This is the current behavior in WinUI:

https://github.com/user-attachments/assets/94989f09-8cde-46f1-91c7-6598f63c9756

As shown, the behavior of `TextBox` and `TextBlock` is not aligned, which is counter to user expectations.
